diktat-ruleset/src/main/kotlin/org/cqfn/diktat/api/DiktatCallback.kt

```diff
@@ -0,0 +1,9 @@
+package org.cqfn.diktat.api
+
+import java.util.function.BiConsumer
+
+/**
+ * Callback for diktat process
+ */
+@FunctionalInterface
+fun interface DiktatCallback : BiConsumer<DiktatError, Boolean>
```

Comment:
Additionally, you can create an extension method to convert a `DiktatCallback` to a `LintErrorCallback` so that `DiktatProcessCommand.ktLintParams()` body looks simpler.